apps/voting/scripts/vote-stats.js

```diff
@@ -48,13 +48,13 @@ module.exports = async (cb) => {
   const voting = Voting.at(appAddress)
 
   const [
-    isOpen,
```

Comment:
I did this because I was hitting the `stack too deep` issue... Based on what I checked, this returned value wasn't being used in the frontend.

Reply:
Sounds good! I'm quite sure we do all of the end date calculations locally in the frontend so no need to worry about it there.

This script is used to generate a CSV file that can be used in data visualization tools, and here as well, we shouldn't need the `isOpen` field.

However, I think it'd be worth introducing another public function to allow another contract on-chain to check for a vote's open status, if we're not hitting the contract size limit yet.

Reply:
Sure. I didn't want to add it to avoid increasing the bytecode, but if that’s not a problem let's go with it